scripts/minici-presubmit-all-tests.sh

```diff
@@ -17,6 +17,7 @@ case $1 in
         export JOB_NAME="odo-minikube-pr-build"
         export SENDQUEUE="amqp.ci.queue.minikube.send"
         export SENDTOPIC="amqp.ci.topic.minikube.send"
+        export EXCHANGE="amqp.ci.exchange.minikube.send"
```

Comment:
Would you mind doing the same for minishift? it will be needed as well:
export EXCHANGE="amqp.ci.exchange.minishift.send"